metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -373,6 +375,10 @@ def aip_metaflow_step(
     ):
         metaflow_configs_new["METAFLOW_USER"] = "aip-user"
 
+    if is_join_step and "METAFLOW_S3OP_NUM_WORKERS" not in os.environ:
+        # AIP-7487: Metaflow joins steps require lots of memory
+        os.environ["METAFLOW_S3OP_NUM_WORKERS"] = "5"
```

Comment:
Can we move this `5` to `aip_constants.py` as `AIP_NUM_WORKERS_DEFAULT`? This way we are not directly changing metaflow core, but make this config easier to find or update in the future.

Reply:
done! thank you!